package.json

```diff
@@ -56,9 +56,9 @@
     "jest-pnp-resolver": "1.0.1",
     "jest-resolve": "23.6.0",
     "pretty-bytes": "^5.1.0",
-    "react": "16.8.0-alpha.1",
-    "react-dom": "16.8.0-alpha.1",
-    "react-test-renderer": "16.8.0-alpha.1",
+    "react": "16.8.1",
+    "react-dom": "16.8.1",
+    "react-test-renderer": "16.8.1",
```

Comment:
I am curious why would we use `^16.8.0` in peer dependencies but lock it to `16.8.1` for dev deps?

Reply:
Peer dependencies is the minimum required version of React for this library to work. However, this can still be `^16.8.1` for dev dependencies. Didn't mean to lock it down but wanted to bump to latest for warnings while doing library dev.